OrbitLinuxTracing/TracerThread.cpp

```diff
@@ -731,13 +731,20 @@ void TracerThread::Run(const std::shared_ptr<std::atomic<bool>>& exit_requested)
   }
 
   // Close the file descriptors.
-  {
+  // Do it in a separate dedicated thread as this can take a long time
+  // with a lot of uprobes and uretprobes.
+  std::thread([tracing_fds = tracing_fds_] {
+    pthread_setname_np(pthread_self(), "close(fd)s");
     ORBIT_SCOPE_WITH_COLOR("Closing file descriptors", orbit::Color::kRed);
```

Comment:
this macro might use some resource which could disappear if main thread no longer exists.